### PR TITLE
[12.0][MIG] account_early_payment_discount

### PR DESCRIPTION
**Pull request by p-tombez**

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Please preserve commit history following technical method explained in https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-12.0

File: account_early_payment_discount/models/account_payment_term.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+from odoo import _, api, fields, models
+
+
+class AccountPaymentTerm(models.Model):
+    _inherit = 'account.payment.term'
+
+    early_payment_discount = fields.Boolean(
+        string="Early Payment Discount",
+        default=False
```
Comment (Nikul-Chaudhary) [inline account_early_payment_discount/models/account_payment_term.py]: @p-tombez no need `default=False`

**Review by Nikul-Chaudhary — CHANGES_REQUESTED** [review]
Improve Code

**Review by leemannd — APPROVED** [review]
Base work seems legit

**Review by i-vyshnevska — APPROVED** [review]

**Comment by tbaden** [comment]
@Nikul-Chaudhary your change was done, anything else or good to go?

**Comment by kittiu** [comment]
@p-tombez please squash your 2 commits into 1 commit.

File: account_early_payment_discount/readme/CONTRIBUTORS.rst [hunk 2]
```diff
@@ -0,0 +1 @@
+* Cyril Gaudin <cyril.gaudin@camptocamp.com>
```
Comment (kittiu) [inline account_early_payment_discount/readme/CONTRIBUTORS.rst:1]: You can add yourself as contributor for this migration.

**Review by kittiu — APPROVED** [review]
This base module looks ready.
Can it be merged now, as it seem many addons is depending on it.

**Pull request merged**